src/components/labels/Labels.tsx

```diff
@@ -30,12 +34,24 @@ export default function Labels(props: LabelsProps) {
     return <>Unable to get labels</>;
   }
 
+  if (loading) {
+    return <Skeleton width="100%" />;
+  }
+
+  if (labels != null && labels.length === 0) {
+    return <>No labels found</>;
+  }
+
   return (
     <>
       {labels.map((label: ImageLabel) => (
-        <Label key={label.key}>
-          {label.key} = {label.value}
-        </Label>
+        <>
+          <Label key={label.key} className="label">
+            <span className="label-content">
+              {label.key} = {label.value}
+            </span>
+          </Label>{' '}
```

Comment:
can you remove the `{' '}`

Reply:
That's to add a space between the labels. Taken from the Patternfly examples - https://www.patternfly.org/v4/components/label#filled